getting-started/default_posture.md

```diff
@@ -0,0 +1,149 @@
+# Default Posture
+
+KubeArmor supports configurable default mode of operations. Default Posture comes into picture when there's atleast one `Allow` policy i.e. KubeArmor is handling policies in whitelisting manner ( More about this in [Consideration in Policy Action](https://github.com/kubearmor/KubeArmor/blob/event-auditor/getting-started/consideration_in_policy_action.md) ).
+
+There are two default mode of operations available `block` and `audit`. `block` mode blocks all the operations that are not allowed in the policy. `audit` generates telemetry events for operations that would have been blocked otherwise.
+
+KubeArmor has 4 types of resources: Process, File, Network and Capabilities. Default Posture is configurable for each of the resources seperately except Process. Process based operations are treated under File resource only.
+## Configuring Default Posture
+
+### Global Default Posture
+
+Global default posture is configured using configuration options provided in KubeArmor binary.
```

Comment:
where can we configure global default posture?
arguments? config file?
would be better to give an example.

Reply:
I followed up with an example using flags, will update it with configuration file.